test/library/packages/Sort/RadixSort/radixMSBModule.chpl

```diff
@@ -0,0 +1,83 @@
+ use BitOps;
+ use Sort;
+ use Random;
+ config param bucketBits=8; 
+
+ module MSBmodule {
+
+   proc radixSortMSB(array:[?dom] int) {
+     var auxArray:[1..array.size] int;
+     var place = findPlace(array);
+     __radixSortMSB(array,place,dom.low,dom.high,auxArray);
+   }
+  
+   private proc __radixSortMSB(array:[] int, place: int, startIndex: int, endIndex: int, auxArray:[] int) {
+
+     param numBuckets = (1 << bucketBits);
+     var counts:[1..numBuckets+1] int;  
```

Comment:
Let's declare counts to start from 0, so we don't have to add 1 all the time: `0..numBuckets+1`